forward/ft_prepare_vol_sens.m

```diff
@@ -533,6 +536,22 @@
         headmodel.elec = sens;
       end
 
+    case 'duneuro'
+      ft_hastoolbox('duneuro', 1);
+      duneuro_license; % show the license (only once)
```

Comment:
this already happens in ft_hastoolbox, right?

Reply:
yes, right, removed here 05bfc64